Core/Config.h

```diff
@@ -144,7 +144,10 @@ struct Config {
 	int iBufFilter; // 1 = linear, 2 = nearest
 	bool bPartialStretch;
 	bool bStretchToDisplay;
-	bool bSmallDisplay;  // Useful on large tablets with touch controls to not overlap the image. Temporary setting - will be replaced by more comprehensive display size settings.
+	int iSmallDisplayZoom;  // Used to fit display into screen 0 = disabled, 1 = auto, 2 = x1, 3 = x2 etc..
+	float fSmallDisplayOffsetX;
+	float fSmallDisplayOffsetY;
+	float fSmallDisplayCustomZoom;
```

Comment:
Do these only make the display smaller?  In the case of 1080p, it's technically making it larger, right?  Theoretically the zoom can go either way?

-[Unknown]